Tools/px_uploader.py

```diff
@@ -569,7 +569,10 @@ def send_reboot(self):
             return False
 
         print("Attempting reboot on %s with baudrate=%d..." % (self.port.port, self.port.baudrate), file=sys.stderr)
-        print("If the board does not respond, unplug and re-plug the USB connector.", file=sys.stderr)
+        if "tty" in self.port.port:
```

Comment:
I don't think this is robust enough. For example `ttyACM0` is USB too. Maybe check if the port starts with `/dev/ttyS`?